pkg/popeye.go

```diff
@@ -377,13 +377,21 @@ func (p *Popeye) dumpPrometheus() error {
 	return pusher.Add()
 }
 
+func (p *Popeye) getClusterName() string {
+	if *p.flags.InClusterName != "" {
```

Comment:
@danibaeyens We should check for nil ptr here prior to the dereference in case the flag is not specified